### PR TITLE
Update WP & UWP bindings

### PR DESCRIPTION
**Pull request by javiergm1983**

### CODE REVIEW

File: bindings/wp8/MegaSDK.cpp [hunk 1]
```diff
@@ -1110,11 +1108,14 @@ void MegaSDK::setLogLevel(MLogLevel logLevel)
     MegaApi::setLogLevel((int)logLevel);
 }
 
-void MegaSDK::setLoggerObject(MLoggerInterface^ megaLogger)
+void MegaSDK::addLoggerObject(MLoggerInterface^ megaLogger)
+{
+    MegaApi::addLoggerObject(new DelegateMLogger(megaLogger));
```
Comment (sergiohs84) [inline bindings/wp8/MegaSDK.cpp]: You need to store the reference passed to `addLoggerObject()`, so you can pass exactly the same instance to `removeLoggerObject()`

**Comment by javiergm1983** [comment]
Requested changes done. Please, review again.

File: bindings/wp8/MegaSDK.cpp (hunk 1 above)
Comment (javiergm1983) [inline bindings/wp8/MegaSDK.cpp]: Requested changes done. Please, review again.

File: bindings/wp8/MegaSDK.cpp [hunk 2]
```diff
@@ -3439,3 +3470,24 @@ void MegaSDK::freeTransferListener(DelegateMTransferListener *listener)
...
+    return delegateLogger;
+}
+
+void MegaSDK::freeLogger(DelegateMLogger *logger)
+{
+    if (logger == nullptr) return;
+
+    EnterCriticalSection(&loggerMutex);
+    activeLoggers.erase(logger);
+    LeaveCriticalSection(&loggerMutex);
```
Comment (sergiohs84) [inline bindings/wp8/MegaSDK.cpp]: Maybe it's not necessary to erase the item from the vector, since it's already erased in `MegaSDK::removeLoggerObject()`

Comment (javiergm1983) [inline bindings/wp8/MegaSDK.cpp]: Requested changes done.

**Comment by javiergm1983** [comment]
Requested changes done.

**Pull request merged**